flexmeasures/api/v3_0/tests/test_assets_api.py

```diff
@@ -219,26 +219,6 @@ def test_alter_an_asset_with_json_attributes(
     assert asset_edit_response.status_code == 200
 
 
-@pytest.mark.parametrize("requesting_user", ["test_admin_user@seita.nl"], indirect=True)
-def test_post_an_asset_with_existing_name(client, setup_api_test_data, requesting_user):
```

Comment:
We should update this test to posting an asset with an existing sibling name.

Reply:
Good point!